cityofaustin/janis
# 3500 pending translation

# Description

A Pending Translation component that appears when a user chooses Spanish. 

Because of how the header is laid out, in order to get the white background to match the xd, I created an empty div that is visible when the message is also visible. 









# Types of changes



# How can this be tested?
Switch language to Spanish. Click around to close it, click the link, and navigate to other pages and see the dropdown doesnt reappear.



 

# Checklist:
